charmcraft.yaml

```diff
@@ -56,6 +68,9 @@ resources:
     description: OCI image for kyuubi
     upstream-source: ghcr.io/canonical/charmed-spark:3.4-22.04_edge
 
+
 requires:
   s3-credentials:
-    interface: s3
\ No newline at end of file
+    interface: s3
+  database:
+    interface: postgresql_client
```

Comment:
**todo** we should probably put a limit on the relations. I don't think we could have more than 1 metastore

Reply:
Updated to `limit: 1`.